.github/ISSUE_TEMPLATE.md

```diff
@@ -1,19 +1,24 @@
-**Rancher Versions:**
-Server:
+**Rancher versions:**
+rancher/server:
+rancher/agent:
```

Comment:
Yep, thought it would be nice as a double check bullet. What should be and what is actually there. Can be removed.